src/services/authStorage.ts

```diff
@@ -0,0 +1,113 @@
+import { LOCALSTORAGE_PREFIX } from 'global/const';
+
+const LOCALSTORAGE_AUTH_STORAGE_KEY = `${LOCALSTORAGE_PREFIX}auth`;
+const LOCALSTORAGE_USER_TOKENS_KEY = `${LOCALSTORAGE_PREFIX}userTokens`;
+
+const AUTH_STORAGE_CURRENT_ACCOUNT = 'account';
+const AUTH_STORAGE_AUTH_KEY = 'token';
+const AUTH_STORAGE_AUTH_REFRESH_KEY = 'refreshToken';
+
+class AuthStorage {
+  private static setAuth(value: string): void {
+    localStorage.setItem(LOCALSTORAGE_AUTH_STORAGE_KEY, JSON.stringify(value));
+  }
+
+  private static getAuth(): any {
```

Comment:
if this would return something typed then all the code would be fully typed and you wouldn't need all those consts
```suggestion
  private static getAuth(): { [s: AuthStoreKeys]: string } | undefined {
```